### PR TITLE
Add some drivers

### PR DESCRIPTION
**Pull request by eldruin**
I noticed some of my drivers were not part of this list.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1141,6 +1144,7 @@ Work in progress drivers. Help the authors make these crates awesome!
 [LM75]: https://crates.io/crates/lm75
 [lp586x]: https://crates.io/crates/lp586x
 [LS010B7DH01]: https://github.com/byronwasti/ls010b7dh01
+[Lsm303agr]: https://crates.io/crates/lsm303agr
```
Comment (jamesmunns) [inline README.md:1147]:
```suggestion
[LSM303AGR]: https://crates.io/crates/lsm303agr
```

**Review by jamesmunns — APPROVED** [review]
Looks good! I'm not sure if this is case sensitive, but just in case.

**Comment by burrbull** [comment]
There should be different categories in drivers section and some WIP label instead of different section.

**Comment by BartMassey** [comment]
@burrbull That sounds like a reasonable change. Want to open an issue and give us a PR to work with? Would love your help.

**Pull request merged**